auto-ipsec/libreswan/README.md

```diff
@@ -93,14 +93,14 @@ To support revocation, you must run a CRL host/listener.  This will host a copy
  
 It will start up a web server and listen for notifications.  Be sure that the cert_crl_dist option in keylime.conf is set to point to this server.
 
-If a node is revoked, the listener will update the CRL.  All the nodes will run their configured revocation actions:
+If a agent is revoked, the listener will update the CRL.  All the agents will run their configured revocation actions:
 
 ```
 local_action_update_crl
 local_action_crashsa
 ```
 
-The first action updates the CRL stored locally.  The second action will update the CRL in the NSS database and then force a rekey with any nodes communicating with the revoked server. It will match the certificate by its x509 distinguished name. Within a second or two, all nodes in the network should stop communicating with the revoked host. 
+The first action updates the CRL stored locally.  The second action will update the CRL in the NSS database and then force a rekey with any agents communicating with the revoked server. It will match the certificate by its x509 distinguished name. Within a second or two, all agents in the network should stop communicating with the revoked host. 
```

Comment:
some ambiguity here again between agent and the machine that it is on

Reply:
Ditto here -- needs to specify that the cloud machines/nodes (not Keylime) will stop communicating over IPsec.  The Keylime agent software never communicates over IPsec to the verifier/tenant.

Reply:
changed to machines:

> The first action updates the CRL stored locally.  The second action will update the CRL in the NSS database and then force a rekey with any machines communicating with the revoked server. It will match the certificate by its x509 distinguished name. Within a second or two,all machines in the network should stop communicating with the revoked host.

Reply:
Yeah that looks better!  Can we update this in racoon and libreswan to your new paragraph?